spot-contracts/contracts/FeePolicy.sol

```diff
@@ -67,42 +61,26 @@ contract FeePolicy is IFeePolicy, OwnableUpgradeable {
     uint8 public constant DECIMALS = 8;
     uint256 public constant ONE = (1 * 10**DECIMALS); // 1.0 or 100%
 
-    /// @dev Using the same granularity as the underlying buttonwood tranche contracts.
-    ///      https://github.com/buttonwood-protocol/tranche/blob/main/contracts/BondController.sol
-    uint256 private constant TRANCHE_RATIO_GRANULARITY = 1000;
-
     uint256 public constant SIGMOID_BOUND = ONE / 100; // 0.01 or 1%
     uint256 public constant SR_LOWER_BOUND = (ONE * 75) / 100; // 0.75 or 75%
     uint256 public constant SR_UPPER_BOUND = 2 * ONE; // 2.0 or 200%
 
-    /// @notice Reference to the perpetual token.
-    IPerpetualTranche public perp;
-
-    /// @notice Reference to the rollover vault.
-    IVault public vault;
-
+    //-----------------------------------------------------------------------------
     /// @notice The target subscription ratio i.e) the normalization factor.
+    /// @dev The ratio above which the system is considered "over-subscribed".
+    ///      Adds a safety buffer to ensure that rollovers are better sustained.
     uint256 public targetSubscriptionRatio;
 
-    /// @notice The percentage fee charged on minting perp tokens.
-    uint256 public perpMintFeePerc;
-
-    /// @notice The percentage fee charged on burning perp tokens.
-    uint256 public perpBurnFeePerc;
+    //-----------------------------------------------------------------------------
 
-    /// @notice The percentage fee charged on minting vault notes.
-    uint256 public vaultMintFeePerc;
+    //-----------------------------------------------------------------------------
+    // Perp fee parameters
 
-    /// @notice The percentage fee charged on burning vault notes.
-    uint256 public vaultBurnFeePerc;
-
-    /// @notice The fixed amount vault fee charged during each deployment.
-    /// @dev Denominated in the underlying collateral asset and
-    ///      Paid by the vault note holders to the system owner.
-    uint256 public vaultDeploymentFee;
+    /// @notice The percentage fee charged on minting perp tokens.
+    uint256 private _perpMintFeePerc;
```

Comment:
Mostly because the public view method `perpMintFeePerc` returns the applied fee based on the sr. We could consider renaming this one to something else and make it public. Any suggestions?

Reply:
we can go back to prefixing the view methods with `compute`